### PR TITLE
Af1 algorithm implementation

### PR DESCRIPTION
**Pull request by samhachem99**
the algorithm is fully implemented and working

### CODE REVIEW

File: SQL/CreateAllTables.sql [hunk 1]
```diff
@@ -30,17 +31,16 @@ CREATE TABLE plan_contains
 
 CREATE TABLE plan
 (
-  plan_id     INT NOT NULL AUTO_INCREMENT,
+  plan_id     INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```
Comment (edwardpassagi) [inline SQL/CreateAllTables.sql]: Why did you change the PRIMARY KEY definition like this? The rest of the codebase declared it like PRIMARY KEY (plan_id)

File: nutrition.py [hunk 2]
```diff
@@ -145,6 +151,13 @@ def link_food_to_meal():
 
 if __name__ == "__main__":
     # do not uncomment the below line unless you are sure of its side effects, 
-    # it will clear out all the data. 
-    processDataIntoDatabase()
+    # it will clear out all the data.
+    # start_time = time.time()
+    # processDataIntoDatabase()
+    # plan: PlanBean = QueriesAction.getPlanTest1()
```
Comment (edwardpassagi) [inline nutrition.py]: remove line 157-162

**Review by edwardpassagi — COMMENTED** [review]
Other changes LGTM

File: SQL/mysql/nutrient_doses.sql [hunk 3]
```diff
@@ -80,9 +80,9 @@ VALUES (1101, 1000, 'F', false, false, 0, 2, 11);
 
 -- Copper Data:
 INSERT INTO nutrient_doses(nutrient_id, max_age, gender, pregnant, nursing, LB, IA, UB)
-VALUES (1098, 18, 'IDC', false, false, 300, 890, 8000);
+VALUES (1098, 18, 'IDC', false, false, 0, 1, 8);
 INSERT INTO nutrient_doses(nutrient_id, max_age, gender, pregnant, nursing, LB, IA, UB)
-VALUES (1098, 1000, 'IDC', false, false, 300, 900, 10000);
+VALUES (1098, 1000, 'IDC', false, false, 0, 1, 10);
```
Comment (stonov) [inline SQL/mysql/nutrient_doses.sql:86]: I see you changed the units to be more consistent with the food database

**Review by edwardpassagi — APPROVED** [review]
LGTM
